### ISSUE TITLE
Commands wrong to limit group creation

### ISSUE THREAD

**Issue by Parroman** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#186]
I tested this and $setting["EnableGroupCreation"] = "true" still allows users to create groups. Need to use $setting["EnableGroupCreation"] = "false".  This article has similar information and more detail on restrictions:
https://support.office.com/en-us/article/manage-who-can-create-office-365-groups-4c46c8cb-17d0-44b5-9776-005fced8e618

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 20423b44-629b-1bee-280c-fb55680cd33e
* Version Independent ID: a60f6a48-89d7-98b6-4245-4be171fd06fd
* Content: [Assign roles and permissions in Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/assign-roles-permissions)
* Content Source: [Teams/assign-roles-permissions.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/assign-roles-permissions.md)
* Service: **msteams**
* GitHub Login: @LolaJacobsen
* Microsoft Alias: **lolaj**

**Comment by kenwith** [comment]
@Parroman - looking into this. Thank you for reporting!

**Comment by kenwith** [comment]
@Parroman - thank you for finding this and submitting as feedback. It is greatly appreciated. The article you linked on support.office.com is more accurate as you mentioned so we combined and linked to it. Just a note that the support.office.com article will be migrating to docs.microsoft.com and GitHub soon too so will be open source. We will look to combine these to articles so information is clearer and in one source of truth. In the meantime the link over to the article you mentioned will really help. Much appreciated!

Here is Pull Request (PR) with fix: https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/248

**Comment by myfope** [comment]
Hi,
I think that the article is wrong. A Team Owner is not able to create teams. This is also stated here: https://support.office.com/en-gb/article/team-owner-capabilities-in-teams-942f9d78-1c25-4201-a631-570086a4cf2b

**Comment by AndreaBarr** [comment]
@myfope Thank you for this feedback

I want to make sure this also gets addressed. I am sending this to the Teams/Skype for Business team to be investigated and updated.

I was able to get this into the backlog for the Teams/Skype for Business team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
